lxd/daemon.go

```diff
@@ -796,7 +806,10 @@ func (d *Daemon) Stop() error {
 	}
 
 	if n, err := d.numRunningContainers(); err != nil || n == 0 {
-		shared.Debugf("Unmounting shmounts: err %s n %d", err, n)
+		shared.Log.Debug(
+			"daemon.Stop: unmounting shmounts",
```

Comment:
Current logging standard is:
 - All strings start with a capital letter
 - No strings end with a dot
 - Function name if useful can be provided through context